Userland/Libraries/LibWeb/Bindings/MainThreadVM.cpp

```diff
@@ -58,9 +60,20 @@ JS::VM& main_thread_vm()
 
         static_cast<WebEngineCustomData*>(vm->custom_data())->event_loop.set_vm(*vm);
 
-        // FIXME: Implement 8.1.5.1 HostEnsureCanCompileStrings(callerRealm, calleeRealm), https://html.spec.whatwg.org/multipage/webappapis.html#hostensurecancompilestrings(callerrealm,-calleerealm)
+        // 8.1.5.1 HostEnsureCanAddPrivateElement(O), https://html.spec.whatwg.org/multipage/webappapis.html#the-hostensurecanaddprivateelement-implementation
+        vm->host_ensure_can_add_private_element = [](JS::Object const& object) -> JS::ThrowCompletionOr<void> {
+            dbgln("checking cna add private for: {}", object.class_name());
```

Comment:
leftover debugging?

Reply:
Oh no, how embarrassing you can see my quickly typed debug messages.